test/e2e/common/node/framework/cgroups/cgroups_linux.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
the name implies that this file will have some build directives

Reply:
Even if I build and run the tests from macOS, it should be targeting linux pods remotely, and should not depend on linux locally where the test binary is running.

IMHO this is a bug in the tests.

Reply:
https://github.com/kubernetes/kubernetes/pull/135739